modules/virt-generalizing-windows-sysprep.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-automating-windows-sysprep.adoc
+
+[id="virt-generalizing-windows-sysprep_{context}"]
+= Generalizing a Windows VM using Sysprep
+
+Generalizing an image will allow that image to remove all system-specific configuration data when deployed on a virtual machine.
+
+Prior to generalizing the VM, you must ensure the `sysprep` tool cannot detect an answer file after the unattended Windows installation.
+
+.Procedure
+
+. Remove the sysprep disk.
+.. In the web console, select *Virtualization* -> *Virtual Machines*, and select the relevant VM.
+.. Click *Disks*.
+.. Click the Options menu {kebab} for the `sysprep` disk, then click *Delete*.
```

Comment:
might sound sily but what is kebab

Reply:
Please review the resolved conversations and the preview link of the content. This is AsciiDoc formatting.